src/GeneratedHydrator/CodeGenerator/Visitor/HydratorMethodsVisitor.php

```diff
@@ -90,6 +90,25 @@ static function (ReflectionProperty $property) : bool {
         ));
     }
 
+    private function generatePropertyHydrateCall(ObjectProperty $property, string $input): array
+    {
+        $ret = [];
+
+        $propertyName = $property->name;
+        $escapedName = \addslashes($propertyName);
```

Comment:
If you need an escaped name, please use `var_export()` instead: no quotes needed afterwards

Reply:
OK, is there a rationale for `var_export()` being better than `addslashes()` ?

As a side note, this is actually an additional, previously it wasn't escaped at all, but better be safe than sorry.